### PR TITLE
Gett 84880 implement fix

### PR DESCRIPTION
**Pull request by d1slike**

### CODE REVIEW

File: dead_pool_reaper_test.go [hunk 1]
```diff
@@ -336,9 +336,8 @@ func TestDeadPoolReaperCleanStaleLocks(t *testing.T) {
 	// now clean lock info for workerPoolID2
 	reaper.cleanStaleLockInfo(workerPoolID2, jobNames)
 	assert.NoError(t, err)
-	// both locks should be at 0
+	// lock should be at 0
 	assert.EqualValues(t, 0, getInt64(pool, lock1))
-	assert.EqualValues(t, 0, getInt64(pool, lock2))
```
Comment (egett) [inline dead_pool_reaper_test.go:341]: is it possible to check the value here? should it be 1?

File: dead_pool_reaper.go [hunk 2]
```diff
@@ -95,14 +95,15 @@ func (r *deadPoolReaper) reap() error {
...
 		}
-		// Remove dead pool from worker pools set
-		if _, err = conn.Do("SREM", workerPoolsKey, deadPoolID); err != nil {
-			return err
-		}
 		// Cleanup any stale lock info
 		if err = r.cleanStaleLockInfo(deadPoolID, lockJobTypes); err != nil {
 			return err
 		}
+
```
Comment (egett) [inline dead_pool_reaper.go]: is empty line needed?

File: dead_pool_reaper_test.go (hunk 1 above)
Comment (egett) [reply dead_pool_reaper_test.go:341]: fixed

File: dead_pool_reaper.go (hunk 2 above)
Comment (egett) [reply dead_pool_reaper.go]: fixed

**Review by egett — APPROVED** [review]

**Review by RattusPetrucho — APPROVED** [review]

**Pull request merged**